<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyHttpDownloadClass</class>
 <widget class="QWidget" name="MyHttpDownloadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyHttpDownload</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>101</width>
     <height>97</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="pButtonStart">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>开始下载</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButtonStop">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>暂停下载</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButtonClose">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>取消下载</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>23</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>下载链接：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="7">
     <widget class="QLineEdit" name="downloadUrl"/>
    </item>
    <item row="1" column="0" colspan="8">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>下载目录：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dirLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pButtonchoice">
        <property name="text">
         <string>选择目录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>下载状态：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string>无下载任务</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="8">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>下载速度：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="labelSpeed">
      <property name="text">
       <string>0 B/S</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>剩余时间：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6" colspan="2">
     <widget class="QLabel" name="labelRemainTime">
      <property name="text">
       <string>0 S</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>当前下载大小：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QLabel" name="labelCurrentDownload">
      <property name="text">
       <string>0 B</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>文件大小：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLabel" name="labelFileSize">
      <property name="text">
       <string>0 B</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="launchCheckBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch file</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>311</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>md5:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="md5_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>launchCheckBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
